File: laboratorio-1/respuestas/php/main.php

<!-- php main.php -->
<?php
echo "laboratorio 1 \n";
function sum(int $x, int $y){
    return $x+$y;
}
function dependWithMe(int $x, int $y){
    if ($x > $y) return $x * $y;
    if ($x < $y) return ($x * $y)-($x + $y);
    return ($x * $y) *2;
}

 $result = sum(2,2);
echo "El resultado de la suma es: ($result) \n";
$resultWithMe = dependWithMe(2,2);
echo "El resultado es: ($resultWithMe) \n";
?>

File: laboratorio-1/respuestas/golang/main.go
package main

import "fmt"

func main() {
	fmt.Println("Laboratorio 1")

	result := sum(2, 3)
	fmt.Println("El resultado de la suma es", result)
	resultWithMe := dependWithMe(2, 3)
	fmt.Println("El resultado es: ", resultWithMe)
}

func sum(x, y int) int {
	return x + y
}

func dependWithMe(x, y int) int {
	if x > y {
		return x * y
	}
	if x < y {
		return (x * y) - (x + y)
	}
	return (x * y) * 2
}
